Packs/ANYRUN/Integrations/ANYRUN/ANYRUN.py

```diff
@@ -215,42 +397,281 @@ def operate_on_dict(the_dict):
         raise TypeError(err_msg)
 
 
-def generate_dbotscore(response):
-    """Creates DBotScore object based on the contents of 'response' argument
+def extract_indicators(response: Dict) -> List[CommandResults]:
```

Comment:
Please split this method for each "if" section you have as we talked in the demo.